content/resources/students/_index.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
this file needs to be added to the English folder @sonali9696 because of updates on the way translations work